eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -6,40 +6,51 @@ parameters:
   DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
   TargetDocRepoOwner: 'not-specified'
   TargetDocRepoName: 'not-specified'
+
+
 stages:
   - stage: Signing
+    variables:
+      - template: /eng/pipelines/templates/variables/globals.yml
+      - template: /eng/pipelines/templates/variables/image.yml
     dependsOn: ${{parameters.DependsOn}}
     jobs:
       - deployment: SignPackage
         environment: esrp
         pool:
           name: azsdk-pool-mms-win-2022-general
-          vmImage: windows-2022
+          image: azsdk-pool-mms-win-2022-1espt
```

Comment:
Should these be using the image.yml variables as well?

Reply:
For a reason I haven't figured out yet, deployment jobs do not work with the stage variables, they will try to run the pool as the variable ref string not the resolved value.